### ISSUE TITLE
'Module loading scheme' not supported for browserify

### ISSUE THREAD

**Issue by TheSavior** [elliotf/mocha-sinon#7]
I'm trying to use mocha-sinon in the browser with browserify where sinon isn't attached to window.

```
if (
    typeof window === "object"
    && typeof window.sinon === "object"
  ) {
    plugin(window.sinon);
  } else if (typeof require === "function" && typeof window === 'undefined') {
    var sinon = require('sinon');

    module.exports = function () {
      plugin(sinon);
    };

    plugin(sinon);
  } else {
    throw "This module loading schema is unsupported.  Pull requests are welcome!";
  }
```

The `else if` above works perfectly if the check for `&& typeof window === 'undefined'` is removed. Is there a particular reason it needs to be there?

**Comment by arcticShadow** [comment]
Hi, 
Im working with @MarshallOfSound and we have encountered the above issue. 

The attached PR removes the extraneous check, allowing mocha-sinon to fucntion in  abrowser, after it has been browserified.

Whats teh chances that we can get this merged and published?

**Comment by elliotf** [comment]
Merged and published as version 1.1.5, thank you for your help!
